src/webapp.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import os
+
+import sentry_sdk
+from flask import jsonify, request, Flask
+
+from .handle_event import handle_event
+
+APP_DSN = os.environ.get("APP_DSN")
+if APP_DSN:
+    # This tracks errors and performance of the app itself rather than GH workflows
+    sentry_sdk.init(
+        dsn=APP_DSN,
+        # Set traces_sample_rate to 1.0 to capture 100%
+        # of transactions for performance monitoring.
+        # We recommend adjusting this value in production.
+        traces_sample_rate=1.0,
+        environment=os.environ.get("FLASK_ENV", "production"),
```

Comment:
In case I decide to test Sentry error reporting from my local dev it will be marked as "development". See `.flaskenv`.